lib/rock_rms/resources/group_member.rb

```diff
@@ -19,6 +19,16 @@ def delete_group_member(id)
         delete(group_member_path(id))
       end
 
+      def create_known_relationship(person_id:, related_person_id:, relationship_role_id:)
+        url_params = {
+          personId: person_id,
+          relatedPersonId: related_person_id,
+          relationshipRoleId: relationship_role_id
+        }
+
+        post("GroupMembers/KnownRelationship?#{URI.encode_www_form(url_params)}")
```

Comment:
Just curious, do you have to use the `URI.encode_www_form` method call?

I was under the impression that Faraday would just magically figure it out if you passed two arguments to `post()`

```ruby
params = {
  personId: person_id,
  relatedPersonId: related_person_id,
  relationshipRoleId: relationship_rold_id
}
post('GroupMembers/KnownRelationship', params)
```

Reply:
faraday magic does figure it out, but it figures out to use form params and Rock 500 errors on it.

what is needed is POST but with the params in the url

Reply:
Gotcha!

Thanks, I learned something today.